tests/test_Consumer.py

```diff
@@ -39,6 +39,20 @@ def dummy_assign_revoke (consumer, partitions):
     partitions = list(map(lambda p: TopicPartition("test", p), range(0,100,3)))
     kc.assign(partitions)
 
+    # Verify assignment
+    assignment = kc.assignment()
+    assert partitions == assignment
+
+    # Get cached watermarks, should all be invalid.
+    lo, hi = kc.get_watermark_offsets(partitions[0], cached=True)
+    assert lo == -1001 and hi == -1001
```

Comment:
Should be capitalized to be pep8 style. `confluent_kafka.OFFSET_INVALID`? `INVALID_OFFSET` reads a bit better, but it might make more sense to just stay consistent with the underlying name.